api/metric/api.go

```diff
@@ -22,60 +22,156 @@ import (
 	"go.opentelemetry.io/api/unit"
 )
 
-type MetricType int
+type Type int
```

Comment:
Minor suggestion from the peanut gallery: pick a word other that "type" for this. GCP uses "kind" so there's maybe some prior art for using that terminology. The word "type" can then be used exclusively to refer to int/float/etc (the value type) and "kind" can refer to the orthogonal idea of how to interpret those values.

Reply:
Thanks, good suggestion.

Reply:
Done.